pkg/securitycontext/provider.go

```diff
@@ -95,3 +96,53 @@ func makeCapabilites(capAdd []api.Capability, capDrop []api.Capability) ([]strin
 	}
 	return addCaps, dropCaps
 }
+
+func determineEffectiveSecurityContext(pod *api.Pod, container *api.Container) *api.SecurityContext {
```

Comment:
the doc fir this feature should handle this corner case and call it out as something not to do